src/LSST/control/ControlMain.cpp

```diff
@@ -57,16 +66,161 @@ ControlMain::Ptr ControlMain::getPtr() {
     return _thisPtr;
 }
 
-ControlMain& ControlMain::get() {
-    if (_thisPtr == nullptr) {
-        throw system::ConfigException(ERR_LOC, "ControlMain has not been setup.");
+ControlMain::ControlMain() {}
+
+ControlMain::~ControlMain() {}
```

Comment:
```
class ControlMain {
private:
    ~ControlMain() = default;
```

Reply:
I had really expect something to be needed in both of those.